contribute/style/grammar.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Grammar and style
+description: Grammar and style guidelines for technical documentation
+keywords: grammar, style, contribute
+toc_max: 2
+--- 
+
+## Acronyms and initialisms
+
+An acronym is an abbreviation you would speak as a word, for example, ROM (for read only memory). Other examples include radar and scuba, which started out as acronyms but are now considered words in their own right.
+
+An initialism is a type of acronym that comprises a group of initial letters used as an abbreviation for a name or expression. If you were using the acronym in a spoken conversation, you would enunciate each letter: H-T-M-L for Hypertext Markup Language.
+
+### Best practice:
+
+- Spell out lesser-known acronyms or initialisms on first use, then follow with the acronym or initialism in parentheses. After this, throughout the rest of your page or document, use the acronym or initialism alone.
+> ‘You can use Single-Sign-on (SSO) to sign in to Notion. You may need to ask your administrator to enable SSO.’
```

Comment:
Question: when we introduce an acronym in parentheses, do we capitalize the spelled-out word, as is done here? Even though we wouldn't normally? (single sign-on)

Reply:
This is actually a mistake in my draft. It should be single-sign on. Good spot! Will amend.